### ISSUE TITLE
Size of database vs size of folder

### ISSUE THREAD

**Issue by phirestalker** [scrubbbbs/cbird#16]
Are there any quick and dirty estimates for how much space the database will need for a given directory of images based on space, or number of files?

I need to know how much space I have to clear to maintain a database on an external drive.

**Comment by scrubbbbs** [comment]
You don't need much.

I was curious so I checked one index I have with 500,000 images:
- dct:  ~ 500 bytes / image
- fdct: ~ 4KB/image
- orb: ~ 14kB/image
- color: ~ 500 bytes/image

**Comment by phirestalker** [comment]
Maybe I forgot how to do math, but wouldn't that be 2 million kilobytes or 2 gigabytes for 100,000 images?
I'm not saying that's too much for all it can do. I'm just wondering if I'm nuts.

**Comment by scrubbbbs** [comment]
> but wouldn't that be 2 million kilobytes or 2 gigabytes for 100,000 images?

Yes. If you only need the dct algo you can cut that to a few megabytes using `-i.algos dct` for example.

**Comment by phirestalker** [comment]
Yes, thank you. According to ChatGPT DCT is better for finding near-duplicates which is what I am concerned with. I have no desire to find images that simply contain the same object. I may miss some that were drastically resized, but I think I can live with that.
